### ISSUE TITLE
Branch with updates for MAPL-2.0 compatibility?

### ISSUE THREAD

**Issue by LiamBindle** [GEOS-ESM/FVdycoreCubed_GridComp#2]
Hi everyone,

Are there branches of `GEOS-ESM/FVdycoreCubed_GridComp` and `GEOS-ESM/GFDL_atmos_cubed_sphere` that are compatible with MAPL-2.0 developments? I'm working on getting [geoschem/gchp_ctm](https://github.com/geoschem/gchp_ctm) off the ground and I'm getting a compiler error in `fvdycore`'s build because the subroutine `MAPL_NCIOOpen` appears to have been removed in MAPL-2.0 (dev branch is `GEOS-ESM/MAPL/develop`).

I found a commit on `GEOS-ESM/GEOSgcm_GridComp` (this one [here](https://github.com/GEOS-ESM/GEOSgcm_GridComp/commit/38ca7e33fb85aa45f10f5eaae4acf7911835d8d6)) that appears to have updated the `FVdycoreCubed_GridComp` and `fvdycore` subdirectories for MAPL-2.0 developments from an internal branch. Would it possible to get those updates on public branches of `GEOS-ESM/FVdycoreCubed_GridComp` and `GEOS-ESM/GFDL_atmos_cubed_sphere`?

Liam

**Comment by tclune** [comment]
@LiamBindle No - I did not create that branch on these new repos yet.    I got bogged down on dealing with the hermes and GFIO dependencies.     

I'll use this ticket as an action item to myself to make these branches.   Probably will happen this morning unless I find a bigger fire as I work through my morning email.

Note that it might be some time before the MAPL-2.0 branches become default for the GCM.   The gatekeepers are drowning in many small structural PR's that nonetheless require careful testing for each.
